src/pynxtools/dataconverter/helpers.py

```diff
@@ -575,110 +575,52 @@ def is_value_valid_element_of_enum(value, elist) -> Tuple[bool, list]:
     return True, []
 
 
-NUMPY_FLOAT_TYPES = (np.half, np.float16, np.single, np.double, np.longdouble)
-NUMPY_INT_TYPES = (np.short, np.intc, np.int_)
-NUMPY_UINT_TYPES = (np.ushort, np.uintc, np.uint)
-# np int for np version 1.26.0
-np_int = (np.integer,)
-np_float = (np.floating,)
-# Not to be confused with `np.byte` and `np.ubyte`, these store
-# an integer of `8bit` and `unsigned 8bit` respectively.
-np_bytes = (np.bytes_,)
-np_char = (
-    np.str_,
-    np.bytes_,
-    np.chararray,
-)  # Only numpy Unicode string and Byte string
-np_bool = (np.bool_,)
-np_complex = (np.complex64, np.complex128, np.cdouble, np.csingle, np.complex_)
+nx_char = (str, np.chararray)
```

Comment:
I don't think the np.chararray makes sense here. It is actually deprecated. Checking an array of strings with the new function agains str actually works.

Reply:
but there is np.character, see here: https://numpy.org/doc/2.1/reference/arrays.scalars.html
```suggestion
nx_char = (str, np.character)
```